src/test/scala/kamon/jmx/collector/SupportedKamonMetricTypesSpec.scala

```diff
@@ -102,22 +101,26 @@ class SupportedKamonMetricTypesSpec extends WordSpec {
 
     var passedMetricName: String = ""
 
-    val counterMetricMock = mock[CounterMetric]
+    val counterMetricMock = {
```

Comment:
I guess this can line can be just replaced with `val counterMetricMock = mock[kamon.metric.Counter]` ?

Reply:
Done

Reply:
Well it's not changed

Reply:
This is an outdated diff...